docs/man/polkitd.xml

```diff
@@ -40,6 +43,12 @@
       whenever an application calls into the service.
     </para>
 
+    <para>
+      Passing the <option>--no-debug</option> to <command>polkitd</command>
+      will discard the messages from usages of <literal>polkit.log()</literal>
```

Comment:
"from usages of <literal>polkit.log()</literal>" is an unnecessary detail. Users do no need to know the code details.
Just saying "Do not pass debug information to logs." is sufficient and targeted to wider audience.